cli/jobs/spark/user-assigned-identity.json

```diff
@@ -0,0 +1,11 @@
+{
+    "properties":{
+    },
+    "location": "<AZURE_REGION>",
```

Comment:
How are the values of these templates for e.g. <AZURE_REGION> getting populated?

Reply:
@harneetvirk there are placeholders in the samples like <STORAGE_ACCOUNT_NAME>, <BLOB_CONTAINER_NAME> and <FILE_SYSTEM_NAME> which user has to replace before running them. 
Do we have a recommendation for this scenario? How will we automate testing for these? What do you think?

Reply:
As discussed, the samples will be excluded from automation run.